healthcheck/core/src/main/java/org/apache/felix/hc/core/impl/executor/async/AsyncHealthCheckExecutor.java

```diff
@@ -138,13 +135,7 @@ private boolean scheduleHealthCheck(HealthCheckMetadata descriptor) {
             AsyncJob healthCheckAsyncJob = null;
             
             if (isAsyncCron(descriptor)) {
-            	
-                try {
-    				healthCheckAsyncJob = new AsyncQuartzCronJob(getAsyncJob(descriptor), quartzCronSchedulerProvider, "job-hc-" + descriptor.getServiceId(), "async-healthchecks", descriptor.getAsyncCronExpression());
-                } catch(ClassNotFoundException|NoClassDefFoundError e) {
-                    LOG.warn("Can not schedule async health check '{}' with cron expression '{}' since quartz library is not on classpath", descriptor.getName(), descriptor.getAsyncCronExpression());
-                    return false;
-                }
+            	healthCheckAsyncJob = new AsyncCronJob(getAsyncJob(descriptor), healthCheckCronScheduler, "job-hc-" + descriptor.getServiceId(), descriptor.getAsyncCronExpression());

```

Comment:
Should be indented 2 spaces to right

Reply:
There was a long discussion earlier to use specific formatting standard for Felix but as far as I got to know, it is not decided and that's why I wrote everything without any formatting applied. In Eclipse, the changes look good but due to formatting standard in GitHub, it looks different.

Reply:
similarly here